cernopendata_client/cli.py

```diff
@@ -161,6 +185,9 @@ def download_files(server, recid, doi, title, protocol, expand):
     \b
     Examples:
       $ cernopendata-client download-files --recid 5500
+      $ cernopendata-client download-files --recid 1 --filter-name 00E081791775.root
```

Comment:
If we change `--filter-name` to do exact match (see above), then please amend example to use full file name.  You can use 5500 which has shorter file names for example.